generator/template.go

```diff
@@ -138,12 +138,17 @@ type {{cleannamelower .TypeName}} struct {
 	config *Config
 }
 
-func (o *{{cleannamelower .TypeName}}) list(ctx context.Context, limit int, offset int, modifiedSince *time.Time) ([]{{cleanname .TypeName}}, error) {
+func (o *{{cleannamelower .TypeName}}) list(ctx context.Context, limit int, offset int, modifiedSince *time.Time, limitDeleted bool) ([]{{cleanname .TypeName}}, error) {
```

Comment:
`limitDeleted` is a bit confusing. Aren't we simply trying to request either deleted or non-deleted records, but not both?